Form-Controls/index.html

```diff
@@ -1,27 +1,85 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>My form exercise</title>
+  <meta name="description" content="Order a t-shirt from a selection of sizes and colours using this form" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+</head>
+
+<body>
+  <header>
+    <h1><u>T-Shirt Order Form</u></h1>
+  </header>
+  <main>
+    <form>
+  
+       
+        <br>
+        <label for="tshirt-size">Choose a T-shirt size: </label>
+        <br>
+        <select id="tshirt-size" name="tshirt-size" required>
+          <option value="" disabled selected>-- Please choose --</option>
+          <option value="xs">X Small</option>
+          <option value="s">Small</option>
+          <option value="m">Medium</option>
+          <option value="l">Large</option>
+          <option value="xl">X Large</option>
+          <option value="xxl">XX Large</option>
+        </select>
+        <br>
+        <br>
+        <fieldset aria-labelledby="colour-label">
+        <legend>Choose a T-shirt Colour: </legend>
+        <label><input type="radio" name="colour" value="green" required>Green</label>
+        <label><input type="radio" name="colour" value="black" required>Black</label>
+        <label><input type="radio" name="colour" value="grey" required>Grey</label>
+        </fieldset>
+        
+        <br>
+        <br>
+        <fieldset>
+          <legend>Your Details</legend>
+          <br>
+          <label for="title">Title: </label>
+          <select id="title" name="title" required>
+            <option value="" disabled selected>-- Please choose --</option>
+            <option value="mr">Mr.</option>
+            <option value="mrs">Mrs.</option>
+            <option value="ms">Ms</option>
+            <option value="mx">Mx</option>
+            <option value="miss">Miss</option>
+            <option value="dr">Dr.</option>
+            <option value="prof">Prof.</option>
+            <option value="rev">Rev.</option>
+          </select>
+          <br>
+          <br>
+          <label for="first-name">First Name: </label>
+          <input type="text" id="first-name" name="first-name" placeholder="e.g. Jane" required pattern="[A-Za-z\s\-]{2,50}">
+          <br>
+          <br>
+          <label for="last-name">Last Name: </label>
+          <input type="text" id="last-name" name="last-name" placeholder="e.g. Doe" pattern="[A-Za-z\s\-]{2,50}" required>
+          <br>
+          <br>
+          <label for="email">Email Address: </label>
+          <input type="email" id="email" name="email" placeholder="e.g. jane@geemail.com" pattern="^[^@]+@[^@]+\.[^@]{2,}$" title="Please enter a valid email address in the format such as: username@domain.com" aria-describedby="email-help" required>
+          
+        </fieldset>
+        <br>
+        <br>
+        <button type="submit" aria-label="submit button">Submit Order</button>
+    </form>
+  </main>
+  <br>
+  
+  <footer>
+    <!-- change to your name-->
```

Comment:
I would probably remove this comment - it was a placeholder instruction which you've already acted on.